export function deepError() {
    throw new Error('find me');
}
